src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/acts/DcirBiologyActExtractor.scala

```diff
@@ -0,0 +1,62 @@
+package fr.polytechnique.cmap.cnam.etl.extractors.acts
+
+import fr.polytechnique.cmap.cnam.etl.events.{BiologyAct, BiologyDcirAct, DcirAct, EventBuilder}
+import fr.polytechnique.cmap.cnam.etl.extractors.dcir.DcirExtractor
+import fr.polytechnique.cmap.cnam.etl.sources.Sources
+import org.apache.spark.sql.{DataFrame, Row}
+
+import scala.util.Try
```

Comment:
Should be the highest on the import order.

Reply:
fixed